data/schemas/keyboard.jsonschema

```diff
@@ -374,6 +374,7 @@
                     }
                 },
                 "driver": {"type": "string"},
+                "led_count": {"$ref": "qmk.definitions.v1#/unsigned_int"},
```

Comment:
Is there any situation in which these would be needed?

Reply:
Maybe? The only case that comes to mind in the short term, when there is inconsistency in led count and layout data. Where existing keyboards potentially rely on the backing array being filled with zeros. But maybe all of these were refactored. No strong feelings against it's removal.

Reply:
Removed logic